src/components/Auth/FormInputGroup/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const FormInputGroup = ({ labelFor, label, inputType, inputId, forgotPassLink }) => {
+  return (
+    <div className="form-input-group">
+      <div className="form-input-group__container">
+        <label htmlFor={ labelFor } className="form-input-group__label">
```

Comment:
@rafayet-monon I think we can omit this parameter `labelFor` out and use `inputId` instead of that.
Because we normally use `for` attribute with the same like `id` attribute for input.

https://www.w3schools.com/tags/att_label_for.asp